inst/python_cmd.m

```diff
@@ -216,7 +216,15 @@
 
 %!test
 %! % int
-%! assert (python_cmd ('return 123456,') == 123456)
+%! r = python_cmd ('return 123456');
+%! assert (r == 123456)
+%! assert (isinteger (r))
```

Comment:
Right but this way I only test it's an integer class in case some backend wants to mix int32 and int64 for some reason.